src/dbt_mcp/config/settings.py

```diff
@@ -85,6 +87,32 @@ class DbtMcpSettings(BaseSettings):
     # Developer settings
     file_logging: bool = Field(False, alias="DBT_MCP_SERVER_FILE_LOGGING")
 
+    def __repr__(self):
```

Comment:
This is just so I can see in the GHActions if it's an error is due to auto-disable or not.  pytest truncates when it shows `DbtMCPSettings(some_stuff="soiefjo", .... , final_arg="aoijf")`